app/src/main/java/woowacourse/paint/mapper/BrushColorMapper.kt

```diff
@@ -0,0 +1,24 @@
+package woowacourse.paint.mapper
+
+import com.now.domain.BrushColor
+import woowacourse.paint.presentation.uimodel.BrushColorUiModel
+
+fun BrushColor.toBrushColorUiModel(): BrushColorUiModel {
+    return when (this) {
+        BrushColor.RED -> BrushColorUiModel.RED
+        BrushColor.YELLOW -> BrushColorUiModel.YELLOW
+        BrushColor.ORANGE -> BrushColorUiModel.ORANGE
+        BrushColor.GREEN -> BrushColorUiModel.GREEN
+        BrushColor.BLUE -> BrushColorUiModel.BLUE
```

Comment:
이 경우도, 색상 종류가 많아진다면 어떻게 될까요? 🤔 (이하 동일)

위 선택한 코드 부분에 대해 예시를 들자면,
```suggestion
        return BrushColorUiModel.values().firstOrNull { it.name == this.name } ?: BrushColorUiModel.RED
```
이런 식으로 코드를 간결하게 표현해볼 수 있을 것 같아요!
꼭 name을 비교하는 방법이 아니어도 방법은 다양하게 있을 것 같네요!

다만, 제안드린 코드를 예시로 들면
name이 같지 않다거나 하는 실수가 발생할 수 있다는 위험성이 있을 것 같아요.
더 좋은 방법을 고민해보시면 좋을 것 같고, 크롱의 생각에 따라 반영 여부를 결정하시면 될 것 같네요! ;)